docs/lib/auth/fragments/common/sms/flows.md

```diff
@@ -0,0 +1,130 @@
+There are a few ways to integrate phone numbers into an Amplify app's sign-in and verification process. 
+- **As a username**: Users login with a username and password where their phone number acts as the username.
```

Comment:
I think people often get confused on the distinctions between using a phone number AS as username and signin in WITH a phone_number.  In this first bullet point we collapse that distinction a bit; but making the differences clear would take a fair amount of space. Maybe to keep this section a little cleaner we could stick to only the SMS stuff (and possibly discuss these distinctions in another section)?